docs/cns.md

```diff
@@ -0,0 +1,110 @@
+# Chain Name Service Draft
+
+The goal of this module is to be a simple registry with the ability to transfer ownership. Auctions and other mechanisms are out of scope and can be implemented alongside or on top of this system (perhaps, in a custodial way).
+
+# State
+
+## Chain name
+
+Chain name contains information about the chain, most importantly, chain ID used to uniquly identify the chain and a list of peer nodes.
+
+Key: `chain-[chainID] -> Chain Name` 
```

Comment:
Why does this key need to be prefixed if this is the only chains are being stored by the registry?

Reply:
This is the prefix from the root of the module key. I think it's good to always have a prefix for a specific entry in the store even if it's the only data for the module.